tooling/v8-snapshot/cache/darwin/snapshot-meta.json

```diff
@@ -4073,13 +4073,11 @@
     "./packages/server/lib/project_utils.ts",
     "./packages/server/lib/remote_states.ts",
     "./packages/server/lib/request.js",
-    "./packages/server/lib/routes-ct.ts",
-    "./packages/server/lib/routes-e2e.ts",
     "./packages/server/lib/routes.ts",
     "./packages/server/lib/saved_state.ts",
     "./packages/server/lib/server-base.ts",
     "./packages/server/lib/server-ct.ts",
```

Comment:
Should `server-ct.ts` be gone too?

Reply:
Yes!

Reply:
I think these will be regenerated automatically anyway, but I'll remove `server-ct`.